### PR TITLE
markdown: more tweaks

### PR DESCRIPTION
**Pull request by robhoes**

### CODE REVIEW

File: ocaml/doc/basics.md [hunk 1]
```diff
@@ -115,4 +115,4 @@ qualifier is `RW` has an RPC accessor associated with it that sets its value:
 Apart from the RPCs enumerated above, some classes have additional RPCs
 associated with them. For example, the `VM` class has RPCs for cloning,
 suspending, starting etc. Such additional RPCs are described explicitly
-in the API reference.
```
Comment (mseri) [inline ocaml/doc/basics.md]: what changed here ??

File: ocaml/doc/wire-protocol.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ These types are mapped onto XML-RPC types in a straight-forward manner:
   `(k -> v) map` type is only valid when `k` is a `string`, `ref`, or
   `int`, and in each case the keys of the maps are stringified as
   above. For example, the `(string -> double) map` containing the mappings
-  _Mike &rarr; 2.3_ and _John &rarr; 1.2_ would be represented as:
+  _Mike -> 2.3_ and _John -> 1.2_ would be represented as:
```
Comment (mseri) [inline ocaml/doc/wire-protocol.md]: Does the arrow here appear properly without being interpreted in weird markdown related ways?

File: ocaml/doc/wire-protocol.md [hunk 3]
```diff
@@ -313,4 +313,4 @@ To retrieve all the VM records in a single call:
 ['OpaqueRef:1', 'OpaqueRef:2', 'OpaqueRef:3', 'OpaqueRef:4' ]
 >>> records['OpaqueRef:1']['name_label']
 'Red Hat Enterprise Linux 7'
-```
```
Comment (mseri) [inline ocaml/doc/wire-protocol.md]: That's another funny change!

File: ocaml/doc/wire-protocol.md (hunk 2 above)
Comment (kc284) [inline ocaml/doc/wire-protocol.md]: I think the html notation in normal (non code-block test) is safer - what is the reason for changing it?

File: ocaml/doc/basics.md (hunk 1 above)
Comment (robhoes) [inline ocaml/doc/basics.md]: Added newline.

File: ocaml/doc/wire-protocol.md (hunk 3 above)
Comment (robhoes) [inline ocaml/doc/wire-protocol.md]: Same :)

Comment (mseri) [inline ocaml/doc/wire-protocol.md]: :P

File: ocaml/doc/basics.md (hunk 1 above)
Comment (mseri) [inline ocaml/doc/basics.md]: :stuck_out_tongue:

File: ocaml/doc/wire-protocol.md (hunk 2 above)
Comment (robhoes) [inline ocaml/doc/wire-protocol.md]: Look on my screen 👀 :)

Comment (kc284) [inline ocaml/doc/wire-protocol.md]: Ok, it seems the output differs per system... No problem. If  I remember well there are some more instances of `&rarr;` in the markdown_backend.ml you may want to change.

Comment (kc284) [inline ocaml/doc/wire-protocol.md]: Given the difference per system we may be safer with `->` after all; or even `&#45;&gt;` presuming the latter looks ok for you (may be worth double-checking the rest of the escaped characters in markdown_backen.ml).

**Pull request merged**